js/client-chat.js

```diff
@@ -93,7 +93,8 @@
 			}
 			this.tabComplete.reset();
 			this.chatHistory.push(text);
-			text = this.parseCommand(text);
+			// make sure commands in multiline inputs are handled correctly
+			text = text.split('\n').map(this.parseCommand, this).filter(function (line) {return (line !== false);}).join('\n');
```

Comment:
Not sure if I understand. Is it to split before each command rather than just by newline? Why?